TaskMarsCompetition/TestMarsCompetition/Tests/TestCertificate.cs

```diff
@@ -0,0 +1,529 @@
+﻿using System;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TestMarsCompetition.Context;
+using TestMarsCompetition.ModelCert;
+
+using TestMarsCompetition.Page;
+using TestMarsCompetition.Pages;
+
+using TestMarsCompetition.Utilities;
+using OpenQA.Selenium.DevTools.V124.Runtime;
+
+
+
+
+
+
+
+
+
+namespace TestMarsCompetition.Tests
+{
+     class TestCertificate: CommonHooks
+    {
+
+        private LoginData loginData;
+        private TestDataCert testDatacert;  
+        private Login login;
+        private CertificatePage certificatePage;
+        private AssertionCert assertions;
+        
+
+        public TestCertificate() {
+
+            testDatacert = JsonReaderCert.ReadTestData("Utilities/TestDataCertificate.json");
+            loginData = testDatacert.LoginData;
+            certificatePage = new CertificatePage();
+            login = new Login();
+            assertions = new AssertionCert();
+            
+
+
+        }
+
+
+        [Test, Order(1), Description("TC_001 Validate if the creation of Certificate is successful.")]
+
+        public void TC_001_CreateANewCertificationRecord()
+        {
+            //Login to the website
+            login.loginPage(loginData.email, loginData.password);
+            //Go to Tc01 in the Json input file
+            var testCase = testDatacert.TestCases.Find(tc => tc.TestCaseId == "Tc01");
+            Thread.Sleep(1000);
+            //Navigate to Certification Tab
+            certificatePage.GoToTab();
+            //Get the input data for Certification
+            var certData = testCase.InputData.certificationData;
+            //Delete Elements if present
+            certificatePage.DeleteAllElements();
```

Comment:
so good that you have thought about clean up but this not correct! 

your delete all data should have once before any test starts 

and then 

when a test run it should only clean up  data that is adding and that to at the end of the test